google-cloud-debugger/lib/google/cloud/debugger/middleware.rb

```diff
@@ -71,6 +70,45 @@ def call env
           # Stop breakpoints tracing beyond this point
           @debugger.agent.tracer.disable_traces_for_thread
         end
+
+        private
+
+        ##
+        # Consolidate configurations from various sources. Also set
+        # instrumentation config parameters to default values if not set
+        # already.
+        #
+        def load_config **kwargs
+          configuration.project_id = kwargs[:project_id] ||
```

Comment:
But if you just add `kwargs[:project] ||` to this line, you can avoid forcing client code to update? This legacy support can be undocumented.